src/interfaces/IERC6538Registry.sol

```diff
@@ -1,5 +1,5 @@
-// SPDX-License-Identifier: UNLICENSED
-pragma solidity ^0.8.0;
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.23;
```

Comment:
This interface is out of date now because it's missing the methods for the domain separator and incrementing the nonce. Let's carefully review to make sure they're aligned.

Reply:
Nice catch. I'd suggest rely on a CI check like the one described above: https://github.com/ScopeLift/stealth-address-erc-contracts/pull/5#discussion_r1447719835

Reply:
Updated the interface to https://github.com/ScopeLift/stealth-address-erc-contracts/blob/26ffbf78d1aa556faaff599f2d6af8bf5f222653/src/interfaces/IERC6538Registry.sol Added the CI check here [26ffbf7](https://github.com/ScopeLift/stealth-address-erc-contracts/pull/5/commits/26ffbf78d1aa556faaff599f2d6af8bf5f222653)